RadarSDK/Include/RadarInitializeOptions.h

```diff
@@ -0,0 +1,17 @@
+//
+//  RadarInitializeOptions.h
+//  RadarSDK
+//
+//  Created by Kenny Hu on 9/10/24.
+//  Copyright © 2024 Radar Labs, Inc. All rights reserved.
+//
+
+
+#import <Foundation/Foundation.h>
+
+@interface RadarInitializeOptions : NSObject
+
+@property (assign, nonatomic) BOOL autoSetupNotificationConversion;
```

Comment:
Is `setup` the right verb here? Should `conversion` be plural? Would `autoLogNotificationConversions` be clearer?